packages/test/test-end-to-end-tests/src/test/sharedInterval.spec.ts

```diff
@@ -243,6 +244,7 @@ describeFullCompat("SharedInterval", (getTestObjectProvider) => {
             };
             const container = await provider.makeTestContainer(testContainerConfig);
             const dataObject = await requestFluidObject<ITestFluidObject>(container, "default");
+            dataObject.context.containerRuntime.setFlushMode(FlushMode.Immediate);
```

Comment:
Can we explain why this test expect immediate mode?

Reply:
To be honest, It was the easy way of getting out of test failures. I've been working in parallel on another change related to this and this line will be removed and the assertion function adjusted to flush ops before actually asserting. I will backport that change here to make things more explicit.